### PR TITLE
Plan export to PDF

### PR DESCRIPTION
**Pull request by vbhavdal**

### CODE REVIEW

**Review by hmpf — COMMENTED** [review]
You'd need to add the link to urls.py, and, and to the api as well. No need to drag the entire thing into the API V2 details-view, just a link. We should probably also do something with the export-button in the webui, maybe in another PR.

File: src/easydmp/plan/views/__init__.py [hunk 1]
```diff
@@ -807,12 +809,21 @@ class GeneratedPlanPlainTextView(AbstractGeneratedPlanView):
...
-    content_type = 'text/plain'
+    content_type = 'application/pdf'
+
+    def get(self, request, *args, **kwargs):
+        super().get(request, *args, **kwargs)
+        html_string = render_to_string(GeneratedPlanHTMLView.template_name, self.get_context_data())
+        result = HTML(string=html_string).write_pdf()
+        response = HttpResponse(content_type=self.content_type)
+        response['Content-Disposition'] = 'inline; filename={}'.format(
+            request.GET.get('filename', '') or '{}.pdf'.format(self.object.pk))
```
[the remark this answers is not in the document]
Comment (hmpf) [reply src/easydmp/plan/views/__init__.py]: Or: `slugify(filename, allow_unicode=True)` would almost make it safe except it lowercases everything and would remove dots and spaces. The regex `r"^[-_,\.\w, ]+$" is what we'd want.

**Comment by vbhavdal** [comment]
@hmpf Can you clarify what you mean by "add link to urls.py"? The view is already there.

**Comment by hmpf** [comment]
It is, I noticed when I switched `url()` over to `path()`. If you can add the endpoint to the API, I can look at adding a link to the web UI.

**Comment by hmpf** [comment]
A dropdown in a dropdown turns out not to be easy, so I just added another entry to the menu.

File: src/easydmp/plan/views/__init__.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (hmpf) [reply src/easydmp/plan/views/__init__.py]: I tested with NUL-bytes on Firefox. If the filename starts with NUL, firefox uses "generated.pdf" for the current code. If the NUL is not the first character, the characters up to but not including the NUL is used. I'll put it on my TODO-list to make a "filename_slugify", but not do anything here.

**Pull request merged**